evennia/utils/tests/test_containers.py

```diff
@@ -0,0 +1,86 @@
+import unittest
+
+from evennia.utils import containers
+from django.conf import settings
+from evennia.utils.utils import class_from_module
+
+_BASE_SCRIPT_TYPECLASS = class_from_module(settings.BASE_SCRIPT_TYPECLASS)
+
+class GoodScript(_BASE_SCRIPT_TYPECLASS):
+  pass
+
+class BadScript:
+  """Not subclass of _BASE_SCRIPT_TYPECLASS,"""
+  pass
+
+class WorseScript(_BASE_SCRIPT_TYPECLASS):
+  """objects will fail upon call"""
+  @property
+  def objects(self):
+    from evennia import module_that_doesnt_exist
+
+class TestGlobalScriptContainer(unittest.TestCase):
+
+  def setUp(self):
+    settings.GLOBAL_SCRIPTS = {}
+
+  def patch_global_scripts(self, adict):
+    settings.GLOBAL_SCRIPTS.update(adict)
```

Comment:
Since this is a helper, mark it 'private', by naming it `_patch_global_script`. Makes things clearer. 
You could also opt to use `mock.patch` here to set set `GLOBAL_SCRIPTS` data only for this test.

Reply:
By using `override_settings`, `setUp()` and `_patch_global_scripts(...)` became useless so I removed them both.